designs/FloatingVersions-In-PMUI.md

```diff
@@ -0,0 +1,103 @@
+
+# Floating Versions in the Package Manager UI
+
+* Status: In Review
+* Author(s): [Martin Ruiz](https://github.com/martinrrm)
+* Issue: [3788](https://github.com/NuGet/Home/issues/3788) PackageReference with a star in version is displayed under the updates tab/incorrectly represented in the installed tab
+
+## Problem Background
+
+Currently the user can specify a version range or a floating version, but the current behavior of the package manager UI has poor support for this, displaying an incorrect installed version. If the user indicates a floating version in any package (eg `5.*`) the PM UI will display the minimum version possible (`5.0.0`) as installed even though the resolved version could be different of the resolved version in the command `dotnet list package`. Furthermore the user cannot specify a version range free hand.
+
+### Current behavior
+
+### Updates Tab
```

Comment:
The spec below talks about more than just the updates tab. 

I think the headers need a bit of work. 

* Project level PM UI
  * Installed tab
  * Updates tab

* Solution level PM UI
  * Installed tab
  * Updates tab.

I don't think you have the updates tab for solution level PM UI. 
Consider adding it, but it's very similar to the installed tab.

Reply:
Ofc the `#### Project File and Solution Explorer` sub-section is good an useful.